tests/console/docker.pm

```diff
@@ -108,6 +108,16 @@ sub run {
     $output = script_output('docker container run tw:saved curl -I google.de');
     die("network is not working inside of the container tw:saved") unless ($output =~ m{Location: http://www\.google\.de/});
 
+    # docker run support --init parameter
```

Comment:
this comment can also be removed.

I would use as a comment: **Using an init process as PID 1**